### PR TITLE
Milestone 4: Desktop and responsive

### PR DESCRIPTION
**Pull request by belalgamal46**
### Milestone 4 Desktop and Responsive
  - Create media queries
  - Update HTML
  - Change image for desktop
  - Update styles.css to be responsive by changing CSS properties

### CODE REVIEW

File: CSS/styles.css [hunk 1]
```diff
@@ -394,3 +435,288 @@ textarea {
...
+  .card-works {
+    width: 100%;
+    height: auto;
+    display: flex;
+    justify-content: space-between;
+    column-gap: 2.2rem;
+    padding: 2.4rem;
+  }
+
+  .card-works:nth-child(2),
```
Comment (christianbiring1) [inline CSS/styles.css]: - [x]  Kindly make sure your cards are responsive and have enough width on the screen of `768px---1024px`. 
As you can see how they look like on your design :point_down: 
Screenshot from 2022-09-29 18-27-27

File: CSS/styles.css [hunk 2]
```diff
@@ -394,3 +435,288 @@ textarea {
...
+
+/* Desktop Size */
+
+@media only screen and (min-width: 1261px) {
+  :root {
+    --Neturals-1: #091e42;
+  }
+
+  body {
+    display: initial;
```
Comment (christianbiring1) [inline CSS/styles.css]: - [ ] Kindly make sure your body also has a similar color to the design provided in Figma.

File: index.html [hunk 3]
```diff
@@ -18,246 +18,311 @@
...
+                or sign-ups required.
+              </p>
+              <div class="tags">
+                <ul class="tag">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
                 </ul>
               </div>
+              <button type="button" class="btn">See Project</button>
```
Comment (christianbiring1) [inline index.html:137]: - [ ] Kindly make sure also you implement the button interactions like `hover`, `disable`, and `active` as the Figma design

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @belalgamal46 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] The design is great :+1: :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: CSS/styles.css [hunk 4]
```diff
@@ -272,21 +272,24 @@ form ul {
   border-top-right-radius: 70px;
   width: 100%;
   height: auto;
+  min-height: 1263px;
   padding: 0 2.4rem;
   margin-top: 11.4rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
```
Comment (Santiago220991) [inline CSS/styles.css]: Kindly make sure that your project sticks as much as possible to the Figma design provided. Please reduce the space between your about section and the contact form.  :eyes :eyes: 

image

File: CSS/styles.css [hunk 5]
```diff
@@ -249,10 +250,9 @@ form ul {
...
 }
 
-.btn:hover {
-  background: #6070ff;
+.btn:active {
+  background: #2230d2;
   box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
-  border-radius: 8px;
   color: #fff;
 }
```
Comment (Santiago220991) [inline CSS/styles.css]: [OPTIONAL] As the [last code reviewer suggested](https://github.com/belalgamal46/Setup-and-mobile-first/pull/4#discussion_r983773217) please make sure that you are adding interactions to the buttons according to Figma template provided. The hover effect means that when you put the cursor over the button it changes its color, this feature is not working. Kindly fix the hover effect on the rest of the buttons in your project. :pray: :pray: 
image

File: index.html [hunk 6]
```diff
@@ -18,246 +18,311 @@
...
+              <a href="#"> About </a>
             </li>
             <li>
-              <a href="#" target="_blank">
-                <img src="./images/handVector.svg" alt="hand-vector-icon" />
-              </a>
+              <a href="#"> Contact </a>
             </li>
           </ul>
         </div>
```
Comment (Santiago220991) [inline index.html]: - [OPTIONAL] Kindly make sure that your links are working and redirect the user to the different sections of your project. :+1: :smiley: :smiley:

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @belalgamal46,
I'm Santiago your code reviewer for this PR. :wave: :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @belalgamal46 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by manq2010** [comment]
Your website looks clean and neat :clap:  Perhaps you could look into reducing the transition time on the effects. :+1:

**Comment by serengia** [comment]
I like how clean your site looks. [OPTIONALLY]: You could perhaps increase the minimum font of the description on your cards to 16px. That way you will improve the readability of the text to everyone including those with poor vision. Again, great job! 

---

**Comment by jodavid444** [comment]
Your website looks good.

**Pull request merged**